packages/cli/src/bin/pipcook-daemon.ts

```diff
@@ -36,13 +41,24 @@ async function start(): Promise<void> {
     if (message.event === 'ready') {
       daemon.disconnect();
       daemon.unref();
-      logger.success(`Pipcook service started on ${message.data.listen}. To open pipboard, please visit https://pipboard.vercel.app`);
+      let serverInfo;
+      // TODO(Feely): the listen field is deprecated, we will remove this
+      if (message.data.listen) {
+        serverInfo = message.data.listen;
+      } else if (message.data.host && message.data.port) {
+        serverInfo = `${message.data.host}:${message.data.port}`;
+      }
+      logger.success(`Pipcook service started on ${serverInfo}. To open pipboard, please visit https://pipboard.vercel.app`);
     }
   });
   daemon.on('exit', async (code: number) => {
     logger.fail(`Pipcook daemon starts failed with code(${code}).`, false);
     // TODO(yorkie): check if this is local mode.
```

Comment:
Could this be removed now?

Reply:
We have not implemented the local mode, should we remove it?

Reply:
Keep it there, just resolve it.